liquidator/liquidator.js

```diff
@@ -173,35 +190,52 @@ class Liquidator {
       }
 
       Logger.info({
-        at: "liquidator",
-        message: "Will withdraw liquidation🤑",
+        at: "Liquidator",
+        message: "Withdrawing liquidation🤑",
         address: liquidation.sponsor,
         id: liquidation.id,
         amount: this.web3.utils.fromWei(withdrawAmount.rawValue)
       });
 
+      // Attach error handler to promise so that Promise.all does not throw an error.
       withdrawPromises.push(
-        withdraw.send({
-          from: this.account,
-          gas: 1500000,
-          gasPrice: this.gasEstimator.getCurrentFastPrice()
-        })
+        withdraw
+          .send({
+            from: this.account,
+            gas: 1500000,
+            gasPrice: this.gasEstimator.getCurrentFastPrice()
+          })
+          .catch(error => {
+            Logger.error({
+              at: "Liquidator",
+              message: `Failed to withdraw liquidation rewards: ${error.message}`,
+              from: this.account,
+              gas: 1500000,
+              gasPrice: this.gasEstimator.getCurrentFastPrice(),
+              error
```

Comment:
We've chatted about this in slack but do we need to put both the error message here as well as the `error.message` from above? would we not also need to key-value this pair like:
```
error:error
```

thoughts?

Reply:
I've had the same issue, as mentioned to you on Slack I have never seen the `error` property get filled out, it is usually printed as `error: {}`. This is why I added the `error.message` into the message.

Regarding your second question, `error:error` and `error` are the same here, because if you just pass in a key without a value, the key gets the name of the variable, so if the variable is named `error`, the key-value will be `error:error`. I base this on empirical observation